microsoft/testsuites/performance/storageperf.py

```diff
@@ -125,6 +133,199 @@ def perf_premium_datadisks_io(self, node: Node, environment: Environment) -> Non
             max_iodepth=64,
         )
 
+    @TestCaseMetadata(
+        description="""
+        This test case uses fio to test performance of nfs server over TCP with
+        VM's initialized with SRIOV network interface.
+        """,
+        priority=2,
+        timeout=TIME_OUT,
+        requirement=simple_requirement(
+            min_count=2,
+            disk=schema.DiskOptionSettings(
+                disk_type=schema.DiskType.PremiumSSDLRS,
+                data_disk_iops=search_space.IntRange(min=5000),
+                data_disk_count=search_space.IntRange(min=12),
+            ),
+            network_interface=Sriov(),
+        ),
+    )
+    def perf_storage_over_nfs_sriov_tcp_4k(self, environment: Environment) -> None:
+        self._perf_nfs(
+            environment,
+            test_case_name="perf_storage_over_nfs_sriov_tcp_4k",
```

Comment:
I find a way to avoid pass the name itself. Please try if it's possible to get the caller's name in the _perf_nfs.
import inspect
inspect.stack()[0][3] or inspect.stack()[1][3]

Reply:
Something like this would work but doesn't it make it more complicated? Also, how about we pass the testname as an argument when the test is called?

Reply:
It can be added in the _perf_nfs, so it doesn't need to pass the test name as a parameter.